snuba/clickhouse/query.py

```diff
@@ -50,14 +60,14 @@ def __init__(
             column_expr(dataset, util.tuplify(colname), query, parsing_context)
             for colname in column_names
         ]
-        select_clause = u"SELECT {}".format(
```

Comment:
it is quite weird this was still around.

Reply:
Oops, this was the change I was referring to here: https://github.com/getsentry/snuba/pull/790#discussion_r382352395 

I wasn't paying attention to the actual diff on the last update and didn't avoid staging it. Oh well.